src/app/api/series/on-the-air/route.ts

```
@@ -0,0 +1,42 @@
+
+        if (!response.ok) {
+            throw new Error('Failed to fetch data');
+        }
+
+        const data = await response.json();
+        // Ensure each show has the required fields and is from recent years
+        const results = data.results
+            .filter((show: any) => {
+                const airDate = show.first_air_date;
+                if (!airDate) return false;
+                const showYear = parseInt(airDate.split('-')[0]);
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)